app/services/search_service.rb

```diff
@@ -26,10 +26,10 @@ class SearchService
   class EmptyQuery < StandardError
   end
 
-  attr_accessor :t
+  attr_accessor :q
```

Comment:
We will need an interim build that accepts both for a zero downtime deployment. Then later drop the old param